scratchpad/__snapshots__/data.test.ts.snap

```diff
@@ -23,7 +23,7 @@ Array [
   Statement {
     "object": Literal {
       "termType": "Literal",
-      "value": "Scratchpad (1970-1-1)",
+      "value": "Scratchpad (1/1/1970)",
```

Comment:
I don't know what the mistake here was, but...

Ambiguous date patterns are problematic everywhere.  `1/1/1970` is unambiguous by chance; `12/10/1970` is not.  `1970-1-1` is troublesome, but `1970-01-01` at least *suggests* that this app's date-based file-naming pattern might be ISO- and xsd-compliant (which I hope it is).